packages/crypto/src/utils.ts

```diff
@@ -53,3 +53,10 @@ export const validateLength = (
 		  )
 		: ok(buffer)
 }
+
+export const buf2hex = function (buffer: ArrayBuffer) {
```

Comment:
We should prolly rename it to make it clear that it also reverses the array (I believe bytes being stored in reverse is specific to uint256, not a JS thing?). Or reverse before calling it

Reply:
Right, perhaps the method would be more clearly abstracted as `toPrivateKeyHex(scalar: UInt256): string`